lib/commentOnClosedIssue.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+var Cesium = require('cesium');
+var Promise = require('bluebird');
+var rp = require('request-promise');
+
+var getUniqueMatch = require('./getUniqueMatch');
+
+var defined = Cesium.defined;
+
+/** Get comments -> regex search -> post comment
+ *
+ * @param {String} jsonResponse GitHub Response
+ * @param {Object} headers Request headers
+ * @return {Promise<http.IncomingMessage>} Response
+ */
+var commentOnClosedIssue = function(jsonResponse, headers) {
+    if (!defined(jsonResponse) || !defined(headers)) {
+        return Promise.reject('jsonResponse is undefined');
+    }
+    var commentsUrl = jsonResponse.issue.comments_url;
```

Comment:
That's how I had it--I can revert it.

My thinking was that `index.js` could stay unaware about the interior details of `jsonResponse`, and just pass it along.